param/parameterized.py

```diff
@@ -855,7 +850,7 @@ def __getattribute__(mcs,name):
 
 
 @add_metaclass(ParameterMetaclass)
```

Comment:
`add_metaclass` is a Python 2 thing. 

https://github.com/holoviz/param/blob/2a974928537519895a814ef9c1027df7b4b6c00e/param/parameterized.py#L284-L292

Reply:
Indeed, I have addressed that in another PR actually https://github.com/holoviz/param/pull/734/files#diff-c8fc4fb68012b66fbf42ad9e0b4714fd96a3a72b83558bc543130d78bd622eb3R889